spec/bowlingScoreSpec.js

```diff
@@ -0,0 +1,117 @@
+describe('BowlingScore', function() {
+  var game = new BowlingScore();
+
+  describe('#addNewRollScore', function() {
+    it('correctly pushes first frame into raw score array', function() {
+      game.addNewRollScore(6);
+      game.addNewRollScore(3);
+      expect(game.rawScores).toEqual([6, 3]);
+    });
+
+    it('correctly pushes second frame into raw score array', function() {
+      game.addNewRollScore(7);
+      game.addNewRollScore(2);
+      game.addNewRollScore(4);
+      game.addNewRollScore(5);
+      expect(game.rawScores).toEqual([6, 3, 7, 2, 4, 5]);
+    });
+
+    it('correctly pushes strike into raw score array', function() {
```

Comment:
You don't necessarily care about this in terms of the functionality of this object. This is more of an internal thing. What you want this object to be able to do is to calculate a total score.